cmds/exp/traceroute/main.go

```diff
@@ -41,6 +41,12 @@ func parseFlags(args []string) (*traceroute.Flags, error) {
 	f.BoolVar(&flags.TCP, "tcp", false, "Use TCP method. Same as -m tcp")
 	f.BoolVar(&flags.UDP, "udp", true, "Use UDP method. Same as -m udp")
 
+	// Check if args are supplied
+	if len(args[1:]) < 1 {
+		f.Usage()
+		return nil, errFlags
+	}
+
 	f.Parse(unixflag.ArgsToGoArgs(args[1:]))
```

Comment:
if you are checking because unixflag.ArgsToGoArgs handles an empty arg list badly, perhaps we should fix that instead? We should not be protecting a buggy package function?